### DISCUSSION TITLE
Add additional loads to house?

### DISCUSSION THREAD

**Discussion by godfreym29** [ulic75/power-flow-card#39] [Ideas]
Would it be possible to add major loads as branches  of the house please?
Such things as Immersion Heaters, Air Source Heat Pumps and Electric Car Chargers.
Possibly make it generic so you can add a sensor and the ability to customise it with an MDI icon.

Thanks

**Comment by ulic75** [comment]
Hey @godfreym29, thanks for the suggestion.  It's something that has been requested.  It may be implemented in the future.  Right now I'm just focusing on tidying things up, making the existing features a bit more flexible and adding missing bits that make it more closely mirror the historic Energy Distribution card.

**Comment by jacerny** [comment]
I also join the call to add more devices with editable parameters

**Comment by Cameleon99** [comment]
Car charger and immersion heater would be great for me 👍

**Comment by Zaphod360** [comment]
Adding my car charger and immersion heater to this would be perfection.

**Comment by Moseby** [comment]
I'm also showing interest in adding an extra consumer (car charger) so it doesn't look like the house is consuming all of it.

**Comment by mr42hh** [comment]
Yes, please let us add stuff like heat pumps etc.

**Comment by ulic75** [comment]
My focus has shifted.  My goal currently with this card is to have feature parity with the official [Energy Distribution](https://www.home-assistant.io/dashboards/energy/#energy-distribution) card while maintaining any additional features that were already added.  Of course I could always change my mind in the future.  But as of right now I will not be adding additional loads as much as that may disappoint people.

**Reply by paddy859** [reply]
Not to be hole picking but does the official energy distribution card not include the feature for individual loads?

**Reply by ulic75** [reply]
No I do not believe that it does.  Only on https://www.home-assistant.io/dashboards/energy/#devices-energy-graph

**Comment by t3lmoazevedo** [comment]
Would be awsome some disruption with the old legacy card while adding extra consuption-loads like
- Water Heat / AC
- Garage (or Electric Vehicle 1 /car, Electric Vehicle 2 /bike)
- Kitchen
etc

**Comment by flixlix** [comment]
I created a fork and changed a very large portion of the codebase. This new Card includes individual devices and more:
https://github.com/flixlix/power-flow-card-plus
It is also available in HACS a default repository, so feel free to check it out!

**Reply by Moseby** [reply]
Looks really good. Will definately try this tonight. Thank you!

**Reply by pbranly** [reply]
Hi congratulation and thanks for your work
Does your card integrates:
- the so needed capability to add gas to electricity in order to have the total ?
- the recalculation of carbon percentage taking into account the part of carbon from gas ?

Thanks 
Phil

**Reply by Cameleon99** [reply]
Looks good, already got it set up with my EV (with battery SoC) and hot water diverter (with water temp)

**Reply by flixlix** [reply]
> Hi congratulation and thanks for your work Does your card integrates:
> 
> * the so needed capability to add gas to electricity in order to have the total ?
> * the recalculation of carbon percentage taking into account the part of carbon from gas ?
> 
> Thanks Phil

I personally don't have anything using gas, so I don't have experience with how it interacts with home.
If you feel this is missing, please create a detailed issue in the linked repository explaining what you would like to have included in this card
